packages/core/test/replay/CustomMask.test.ts

```diff
@@ -0,0 +1,58 @@
+import { beforeEach, describe, expect, it } from '@jest/globals';
+
+describe('CustomMask', () => {
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  it('returns a fallback when native view manager is missing', () => {
+    jest.mock('react-native', () => ({
+      UIManager: {},
+      View: jest.fn(),
+    }));
+
+    const { Mask, Unmask } = require('../../src/js/replay/CustomMask');
+
+    expect(Mask).toBeDefined();
+    expect(Unmask).toBeDefined();
```

Comment:
Could we also validate that warn('Mask') / warn('Unask'); were called to make sure they are actually the fallback?
Same for the below tests with fallback.